.github/workflows/coverage.yml

```diff
@@ -33,6 +33,10 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}
 
+    # TODO(Crissman): Remove click after issue with allennlp import resolved
```

Comment:
In the TODO comment, would you mind adding the link to this PR so that other contributors can grasp the background of "issue with allennlp import"?